docs/deploying.md

```diff
@@ -369,8 +372,159 @@ support.
 		* Closing connection 0
 		curl: (60) SSL certificate problem: self signed certificate
 		More details here: http://curl.haxx.se/docs/sslcerts.html
+
+## Deploy a v2 registry with Nginx and basic authentication
+
+This example shows you how to securely deploy a v2 registry using basic authentication.
+
+### Install Docker Compose
+
+1. Open a new terminal on the host with your `distribution` directory.
+
+2. Get the `docker-compose` binary.
+
+		$ sudo wget https://github.com/docker/compose/releases/download/1.3.0/docker-compose-`uname  -s`-`uname -m` -O /usr/local/bin/docker-compose
+
+	This command installs the binary in the `/usr/local/bin` directory. 
+	
+3. Add executable permissions to the binary.
+
+		$  sudo chmod +x /usr/local/bin/docker-compose
+
+### Do some housekeeping
+
+1. Remove all the unused images from your local environment:
+
+     If you are building other images and haven't pushed them yet, publish them before
+     running this command.
+
+        $ docker rmi -f $(docker images -q -a )
+    
+     This step is a housekeeping step. It prevents you from mistakenly picking up
+     an old image as you work through this example.
+		
+2. Edit the `distribution/cmd/registry/config.yml` file and remove the `tls` block.
+
+	If you worked through the previous example, you'll have a `tls` block. To use
+	TLS with the registry and Nginx, you should terminate the TLS on the Nginx
+	rather than the registry.
+
+### Configure your system
+
+1. Change to the `distribution/contrib/basicauth` directory.
+
+	This directory contains a single `docker-compose.yml` configuration file and an `nginx`
+	configuration directory. The `nginx` directory contains:
 	
-## Configure Nginx with a v1 and v2 registry
+	| File                      | Description                                                                        |
+  |---------------------------|------------------------------------------------------------------------------------|
+  | `Dockerfile`              | Describes the Nginx image with a configuration that supports basic authentication. |
+  | `docker-registry-v2.conf` | Specifies the headers required by the registry.                                    |
+  | `registry-basic.conf`     | Specifies the basic authentication configuration required by the registry.         |
+  | `registry.conf`           | Specifies the base configuration, including TLS, required by the registry.         |
+  | `ssl`                     | Contains pre-generated certificates to use in the example.                         |
+  | `test.pass`               | Contains a username-password combination to use in testing.                        |
+			
+2. Create a `certs.d` directory on your local machine. 
```

Comment:
Use "docker host" rather than local machine. (Did this work if running from the mac?)